Formula/coin3d.rb

```diff
@@ -111,12 +114,12 @@ def install
     system ENV.cc, "test.cpp", "-L#{lib}", "-lCoin", *opengl_flags, "-o", "test"
     system "./test"
 
-    xy = Language::Python.major_minor_version Formula["python@3.10"].opt_bin/"python3"
+    xy = Language::Python.major_minor_version Formula["python@3.10"].opt_bin/python3
     ENV.append_path "PYTHONPATH", "#{Formula["pyside@2"].opt_lib}/python#{xy}/site-packages"
     # Set QT_QPA_PLATFORM to minimal to avoid error:
     # "This application failed to start because no Qt platform plugin could be initialized."
     ENV["QT_QPA_PLATFORM"] = "minimal" if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
-    system Formula["python@3.10"].opt_bin/"python3", "-c", <<~EOS
+    system Formula["python@3.10"].opt_bin/python3, "-c", <<~EOS
```

Comment:
Since we call `python3` and `python3.10` now, I wonder if we even need to qualify it with `Formula["python@3.10"].opt_bin`. It doesn't hurt, but it's not really needed as much anymore.

It does help if you did `brew test` after `brew unlink python@3.10`, but that's a pretty remote possibility.